src/model/Guess.java

```diff
@@ -11,24 +11,24 @@
  * @author nathan
  */
 public class Guess {
-    public String name;
-    public String typing;
-    public String paradigm;
-    public String level;
-    public int year;
+    private String name;
+    private String typing;
```

Comment:
Currently the program doesn't compile because you removed the `private String level;` if you add that back, everything works fine.